client/service/src/logging.rs

```diff
@@ -0,0 +1,593 @@
+// This file is part of Substrate.
```

Comment:
This file should be really moved to its own crate probably.

Reply:
I think when tracing's API will be improved this file will actually get much smaller (I don't know if that was your concern).

Reply:
I know that it will get smaller. However I don't see why this should be part of the service crate?

Reply:
I don't know... it makes sense to me but I can't really explain.

Where would you put it? In a new crate called "sc-logging"?

Reply:
Something like that. This way we could probably also split up the file a little bit.

Reply:
Fixed in https://github.com/paritytech/substrate/tree/e49e1810edd90f19e312a5507fb467d5f51eee54/client/logging/src